_posts/blog/2018-09-12/2018-09-12-how-to-get-more-buyers-on-openbazaar-with-categories-and-tags.markdown

```diff
@@ -0,0 +1,145 @@
+---
+title: "How To Get <span class='post-title-extra'>More Buyers</span> on OpenBazaar with <span class='post-title-extra'>Categories and Tags</span>"
+layout: post
+date: '2018-09-12 00:30:00 -0300'
+social_title: 'How To Get More Buyers on OpenBazaar with Categories and Tags'
+social_description: 'A little extra work on your OpenBazaar store can give buyers an exceptional experience to make them want to visit again and again! Here is how to use tags and categories to help users find and browse your store with ease.'
+social_image: OpenBazaar demo store - How To Get More Buyers on OpenBazar with Categories and Tags.png
+---
+
+_by Jason Hotelling, OpenBazaar Front End Developer_
+
+Hello! We hope you're having a great time selling and sharing on the OpenBazaar platform. 
+
+OpenBazaar stores boast the fastest setup time of any e-commerce platform. With very little effort, you can set up your store and add listings in minutes. Then, a little extra work can give buyers an exceptional experience to make them want to visit again and again! 
+
+Today we are going to talk about how to use tags and categories to help users find and browse your store with ease.
+
+### Before We Get Started
+
+If you haven't already, please follow our guides to get started with OpenBazaar. They contain instructions and tutorials that span from _Download_ through _Browsing_ and _Selling_.
+
+[The Beginner's Guide to Buying Goods, Services and Cryptocurrency on Openbazaar](https://openbazaar.org/blog/The-Beginners-Guide-to-Buying-Goods-Services-and-Cryptocurrency-on-OpenBazaar/)
+
+If you are new, we also recommend watching our [OpenBazaar 2.0 Tutorial](https://www.youtube.com/watch?v=VzlerMJMOu8) on YouTube.
```

Comment:
I think it would be more clear to say "If you are new to OpenBazaar", so it's clear what "new" is referring to.

Or "If you are new here" if you want to be more folksy.